Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG LIBNODE_VERSION=102
 COPY ./.docker/scripts/entrypoint.sh /root/
 
 RUN apt-get update && \
-    apt-get install -y wget libatomic1 libc-bin && \
+    apt-get install -y wget jq gnupg libatomic1 libc-bin apt-transport-https && \
```

Comment:
You may also remove the whole `jq` package since it is not used anywere.

---

`jq` nowadays is used in https://github.com/eisengrind/docker-altv-server/blob/master/.github/workflows/default.yml for fetching the latest alt:V version parameters in CI.